### ISSUE TITLE
`for-each`  will incorrectly remove the first statement

### ISSUE THREAD

**Issue by pionxzh** [lebab/lebab#353]
### Input
```js
for (let i = 0; i < array.length; i++) {
    const x = 0;
    const item = array[i];
    console.log(item);
}
```

### Actual Output

```js
array.forEach(item => {
    const item = array[i];
    console.log(item);
}
```

Caused error `SyntaxError: Identifier 'item' has already been declared`

### Expected Output

```js
array.forEach(item => {
    const x = 0;
    console.log(item);
}
```

**Comment by nene** [comment]
Thanks for reporting. Looks like Lebab tries to always remove the first variable definition, which is clearly not correct.

Interestingly the `for-of` transform does not have this bug. It just skips the transform when the variable isn't the first one in for-loop. Should at least do the same in case of `for-each` transform.

**Comment by nene** [comment]
Fixed in 3.2.4.

It now only performs the transform when it detects a suitable variable declaration as the first statement of for loop body.

**Comment by pionxzh** [comment]
Thanks for the quick fix!
